docs/scripts_md/batch_image_uploader.md

```diff
@@ -0,0 +1,69 @@
+# NAME
+
+batch\_uploads\_imageuploader -- a script that runs imaging\_upload\_file.pl in
+batch mode
+
+# SYNOPSIS
+
+./batch\_uploads\_imageuploader -profile prod < list\_of\_scans.txt >log\_batch\_imageuploader.txt 2>&1 \`\[options\]\`
+
+Available options are:
+
+\-profile      : name of the config file in
+                `../dicom-archive/.loris_mri`
+
+\-verbose      : if set, be verbose
+
+# DESCRIPTION
+
+This script runs the Loris-MRI insertion on multiple scans. The list of scans
```

Comment:
add the word "scripts" after "insertion"

Reply:
I used pipeline instead to avoid using scripts twice in the same sentence